add_column_snowflake_ds/deploy/0.0.1/ci-deploy.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+set -e
+
+tb deploy --populate --fixtures --wait
```

Comment:
both ci-deploy and cd-deploy are the default ones, so they are not needed, can you try to remove them? (also .tinyenv is probably not necessary to bump version)

Reply:
Removed